### PR TITLE
parse_type 0.6.2 (ursaverance effort)

### PR DESCRIPTION
**Pull request by lorepirri**
[PKG-2696] parse_type 0.6.2

dev_url: https://github.com/jenisys/parse_type 
conda-forge: https://github.com/conda-forge/parse_type-feedstock
dependencies: https://github.com/jenisys/parse_type/blob/v0.6.2/setup.py

- linter
- update to version `0.6.2`, update sha
- remove `noarch`
- python script
- test commands `pip check`

**Dependency for** 
- AnacondaRecipes/behave-feedstock#2

**Depends on:** 
- AnacondaRecipes/parse-feedstock#2

[PKG-2696]: https://anaconda.atlassian.net/browse/PKG-2696?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,31 +1,36 @@
...
   host:
-    - python >=3.6
+    - python
     - setuptools
     - pip
+    - wheel
   run:
-    - python >=3.6
+    - python
     - parse >=1.6
```
Comment (cbouss) [inline recipe/meta.yaml]:
```suggestion
    - parse >=1.18.0
```
https://github.com/jenisys/parse_type/blob/v0.6.2/pyproject.toml#L46

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,31 +1,36 @@
...
   url: https://pypi.io/packages/source/p/parse_type/parse_type-{{ version }}.tar.gz
-  sha256: 20b43c660e48ed47f433bce5873a2a3d4b9b6a7ba47bd7f7d2a7cec4bec5551f
+  sha256: 79b1f2497060d0928bc46016793f1fca1057c4aacdf15ef876aa48d75a73a355
 
 build:
-  number: 1
-  noarch: python
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  number: 0
+  skip: true  # [py37]
```
Comment (cbouss) [inline recipe/meta.yaml]:
```suggestion
```
upstream support is even from python 3.2.

File: recipe/meta.yaml [hunk 3]
```diff
@@ -1,31 +1,36 @@
...
-    - python >=3.6
+    - python
     - setuptools
     - pip
+    - wheel
   run:
-    - python >=3.6
+    - python
     - parse >=1.6
     - six
```
Comment (cbouss) [inline recipe/meta.yaml]:
```suggestion
    - six >=1.15
```
https://github.com/jenisys/parse_type/blob/v0.6.2/pyproject.toml#L49

File: recipe/meta.yaml [hunk 4]
```diff
@@ -1,31 +1,36 @@
...
 test:
   imports:
     - parse_type
+  requires:
+    - pip
+  commands:
+    - pip check
 
 about:
   home: https://github.com/jenisys/parse_type
```
Comment (cbouss) [inline recipe/meta.yaml:35]: LICENSE it MIT
https://github.com/jenisys/parse_type/blob/v0.6.2/LICENSE

**Review by cbouss — CHANGES_REQUESTED** [review]

File: recipe/meta.yaml (hunk 4 above)
Comment (lorepirri) [reply recipe/meta.yaml:35]: fixed, thank you also for the other comments!

**Review by cbouss — APPROVED** [review]

**Review by skupr-anaconda — APPROVED** [review]

**Pull request merged**